pywinauto/windows/uia_element_info.py

```diff
@@ -280,52 +303,70 @@ def parent(self):
         else:
             return None
 
-    def _get_elements(self, tree_scope, cond=IUIA().true_condition, cache_enable=False):
-        """Find all elements according to the given tree scope and conditions"""
+    # def _get_elements(self, tree_scope, cond=IUIA().true_condition, cache_enable=False):
+    #     """Find all elements according to the given tree scope and conditions"""
+    #     try:
+    #         ptrs_array = self._element.FindAll(tree_scope, cond)
+    #         return elements_from_uia_array(ptrs_array, cache_enable)
+    #     except(COMError, ValueError) as e:
+    #         print(e)
+    #         ActionLogger().log("COM error: can't get elements")
+    #         return []
```

Comment:
I think, the unused code should be removed, instead of just commenting it of.

Reply:
done

Reply:
I think so, if we stop using it now.

Reply:
We haven't decided it finally. Maybe it's worth to keep this code to allow enabling it as an option. Also it's useful for performance comparisons. Maybe it will lead to some trade-offs between `FindAll` and `RawViewWalker` approaches.